rmc/pipeline/two_breaks/run_batches.py

```diff
@@ -628,17 +628,27 @@ def main(args):
         default_python_image=args.docker_image,
         requester_pays_project=args.google_project,
     )
+    # Check if user specified list of numbers for batches
+    # These numbers are used to write output files for batch jobs
+    if args.counter:
+        count_list = args.counter.split(",")
 
     if args.under_threshold:
         section_groups = [
             sections_to_run[x : x + args.group_size]
             for x in range(0, len(sections_to_run), args.group_size)
         ]
         count = 1
+
         for group in tqdm(section_groups, unit="section group"):
-            logger.info("Working on group number %s...", count)
+            if count_list:
+                group_num = count_list[count - 1]
```

Comment:
`group_num` ends up being a string instead of an int when passed to `process_section_group` which I think ultimately doesn't matter but just fyi as that's in the specification